Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,29 @@
+class GitWhenMerged < Formula
+  desc "Find where a commit was merged in git."
+  homepage "https://github.com/mhagger/git-when-merged"
+  url "https://github.com/mhagger/git-when-merged/archive/v1.0.0.tar.gz"
+  sha256 "1be0f2c660e6e0cef66593aac681163b97027a7be028b6d2b3901fee00639ad7"
+  bottle :unneeded
+  
+  def install
+    bin.install "bin/git-when-merged"
+  end
+
+  test do
+    system "#{bin}/git-when-merged", "-h"
+    system "git", "init"
+    system "touch", "foo"
+    system "git", "add", "foo"
+    system "git", "commit", "-m", "'foo'"
```

Comment:
You don’t need to add quotes around `foo`: `"foo"`. Otherwise `git` will search for the file `'foo'`.